apps/chef_db/src/chef_db.erl

```diff
@@ -230,6 +230,11 @@ fetch(ObjectRec, #context{reqid = ReqId}) ->
                       not_found |
                       {error, term()}.
 fetch_multi(RecModule, #context{reqid = ReqId}, QueryName, QueryParams) ->
+    %% FIXME: 
```

Comment:
What's the preferred way to fix this? Looks like stats hero always expects a 2-tuple here, so I could just remove the query name from the args. Alternatively I could do something more invasive to try to preserve the QueryName info into stats hero, or use a literal binary to specify my own label (but that spreads the label logic out in a way we probably wouldn't know to look for in a hypothetical future refactoring).